src/custom/constants/index.ts

```diff
@@ -97,3 +97,6 @@ export const GAS_FEE_ENDPOINTS = {
 }
 
 export const UNSUPPORTED_TOKENS_FAQ_URL = '/faq#what-token-pairs-does-cowswap-allow-to-trade'
+
+// fee threshold - should be greater than percentage, show warning
+export const FEE_SIZE_THRESHOLD = new Fraction(30, 100)
```

Comment:
Question: why `Fraction` and not `Percent`?

Not saying it should be `Percent`, just don't really know which should be used.

Reply:
Fraction because the maths calls for it. 

`trade.inputAmount.asFraction` plays nicer here